aea/protocols/default/__init__.py

```diff
@@ -18,3 +18,8 @@
 # ------------------------------------------------------------------------------
 
 """This module contains the support resources for the default protocol."""
+
+from aea.protocols.default.message import DefaultMessage
+from aea.protocols.default.serialization import DefaultSerializer
+
+DefaultMessage.serializer = DefaultSerializer
```

Comment:
Since the `serializer` is a class var; this is sufficient to expose the serializer everywhere.